app/src/main/kotlin/com/wire/android/ui/conversation/mention/model/Mention.kt

```diff
@@ -0,0 +1,17 @@
+package com.wire.android.ui.conversation.mention.model
+
+import com.wire.android.ui.conversation.all.model.Conversation
+import org.koin.ext.quoted
+
+data class Mention(
+    val mentionInfo: MentionInfo,
+    val conversation: Conversation
+)
+
+data class MentionInfo(val mentionMessage: MentionMessage)
+
+data class MentionMessage(val message: String) {
+    fun toQuote(): String {
+        return message.replace("\"", "")
```

Comment:
I don't quite get this one. Why removing the double quotes when displaying the items?
And maybe the naming is a bit counter-intuitive: `toQuote` removes the quotes?

Maybe it could be named `removingQuotes` or something.

Reply:
damn it it was supposed to add the qoutes to the message ;p